octavia_f5/restclient/bigip/bigip_restclient.py

```diff
@@ -117,6 +119,7 @@ def update_status(self):
         return self._active
 
     @_metric_get_exceptions.count_exceptions()
+    @injectF5OSAHeaders
```

Comment:
It is not obvious that the parameters change on the fly. It is better to make a method inside the class for adding headers and call it in the necessary places. One line is also added, like decorators, but this is much clearer and more obvious.